ada_feeding/ada_feeding/trees/move_to_tree.py

```diff
@@ -31,24 +31,16 @@ class MoveToTree(ActionServerBT, ABC):
 
     def __init__(
         self,
-        action_type_class_str: str,
     ) -> None:
-        """
-        Initializes tree-specific parameters.
-
-        Parameters
-        ----------
-        action_type_class_str: The type of action that this tree is implementing,
-            e.g., "ada_feeding_msgs.action.MoveTo". The input of this action
-            type can be anything, but the Feedback and Result must at a minimum
-            include the fields of ada_feeding_msgs.action.MoveTo
-        """
-        # Import the action type
-        self.action_type_class = import_from_string(action_type_class_str)
+        # Uninitialized attributes
+        self.action_type_class = None
+        self.action_type_class_str = ""
```

Comment:
Looks like these two initializations are not necessary -- I checked all files that subclass this, and they only access `self.action_type_class_str` in `create_move_to_tree`, which is called after these parameters are initialized in `create_tree`.

Was this for the linter? See my above comment re. that. My marginal preference is to not unnecessarily initialize variables in `__init__`, but I don't feel super strongly about it so am also okay with this option if you prefer. We should just be on the same page to have a uniform coding style in this repo.

Reply:
I thought it was for the linter, but it's not complaining, removed.